### PR TITLE
README: Add a wiki section in the read me for technical contributors

### PR DESCRIPTION
**Pull request by kdaud**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ OpenMRS [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development) QA fram
 - `mvn clean install -DskipTests=true`
 
 ## Configuration
-Set Your test configurations in `qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties`.
+Set your test configurations in `qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties`.
```
Comment (sherrif10) [inline README.md:79]: Hey @kdaud point of clarification. Did we confirm that our tests are breaking from upgrade of platform version. My take  is it possible to revert back our platform version we depending on and simply depend on the version that is working for us. cc @dkayiwa

Comment (dkayiwa) [reply README.md:79]: @sherrif10 the solution is to fix the tests because the platform has no problem

Comment (dkayiwa) [reply README.md:79]: FYI: https://github.com/openmrs/openmrs-contrib-qaframework/commit/13e8266bc24843f304d930a279a3c5a80a378277

Comment (sherrif10) [reply README.md:79]: @dkayiwa  thanks for the quick fix. that seemed out of scope. Now lets embark on why tests are failing on ci.

Comment (dkayiwa) [reply README.md:79]: https://issues.openmrs.org/browse/REPORT-886

Comment (sherrif10) [reply README.md:79]: Yes @dkayiwa Thanks that is a fantastic quick fix.

**Pull request merged**